assets/src/components/form/base/form-info-base.js

```diff
@@ -56,19 +56,7 @@ const FormInfoBase = ( {
 			onClick={ onDismiss }
 		/>
 	) : null;
-	if ( ! isEmpty( formInfoVars ) ) {
-		const formInfoText = [];
-		const chunks = formInfo.split( '%%var%%' );
-		for ( let x = 0; x < chunks.length; x++ ) {
```

Comment:
Why not `chunks.forEach()`?

Reply:
you want me to edit code that's being removed ;)

j/k

Reply:
Oh my bad 🤦🏻‍♂️